package.json

```diff
@@ -64,7 +64,7 @@
     "tt-wdio-webpack-obj": "node ./bin/wdio-runner.js wdio-runner-cli.js --suite opinionated",
     "tt-wdio-webpack-func": "TT_TEST_WDIO_FUNCTION=true node ./bin/wdio-runner.js --suite opinionated",
     "tt-wdio-unopinionated": "TT_TEST_WDIO_FUNCTION=true node  ./bin/wdio-runner.js --formFactors=['tiny','huge'] --locales=['en','fr'] --suite unopinionated;",
-    "tt-wdio-no-formFactor": "npm run pack && node ./bin/wdio-runner.js --spec tests/wdio/validateElement-spec.js && rm -rf ./build",
+    "tt-wdio-no-formFactor": "npm run pack && node ./bin/wdio-runner.js --spec tests/wdio/validateElement-spec.js --spec tests/wdio/defaultViewports-spec.js && rm -rf ./build",
```

Comment:
Only one spec will be run at a time. Are both tests being run with this change?

Reply:
Yeah, my bad.
I changed it now to a suite including both the spec files.